engine/consensus/approvals/veryfing_assignment_collector.go

```diff
@@ -16,7 +16,7 @@ import (
 	"github.com/onflow/flow-go/state/protocol"
 )
 
-// DefaultEmergencySealingThreshold is the default number of blocks which indicates that ER should be sealed using emergency
+// DefaultEmergencySealingThreshold is the default number of Blocks which indicates that ER should be sealed using emergency
```

Comment:
### ⚠️ 

I assume that capitalizing "Blocks" and "State" in this file was an unintended by-product of exporting the equally-named fields in `engine/consensus/approvals/testutil.go` (?)

**Would suggest to revert all changes to `engine/consensus/approvals/veryfing_assignment_collector.go`**